style.css

```diff
@@ -1,8 +1,18 @@
 /*Typography*/
+@media (min-width:400px){
```

Comment:
This wraps all your CSS in a media query, so there is not styling if the user's screen is narrower than 400px. Is that what you wanted?